Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.yml

```
@@ -91,6 +105,106 @@ script:
   script: ''
   type: python
   commands:
+  - name: argus-add-attachment
+    arguments:
+    - name: case_id
+      required: true
+      description: ID of Argus case
+    - name: file_id
+      required: true
+      description: ID of attachement
+    outputs:
+    - contextPath: Argus.Attachment.responseCode
+      description: API response metadata, response code of this request
+      type: Number
+    - contextPath: Argus.Attachment.limit
+      description: API response metadata, limit of results this request ran with
+      type: Number
+    - contextPath: Argus.Attachment.offset
+      description: API response metadata, the offset into the result-set of this query
+      type: Number
+    - contextPath: Argus.Attachment.count
+      description: API response metadata, total number of results this query has
+      type: Number
+    - contextPath: Argus.Attachment.size
+      description: API response metadata, the number of results returned in this request
+      type: Number
+    - contextPath: Argus.Attachment.messages.message
+      description: Attachment Messages Message
```

```suggestion
      description: Attachment Messages Message
```

Can you provide a better description here? what is messages message?

---

This has been approved before, and I do not have the time to change this now. I would appreciate if we could focus on the changes in this PR, unless there are critical issues.